inc/rccl.h

```diff
@@ -190,6 +190,23 @@ rcclResult_t rcclReduce(const void* sendbuff, void* recvbuff, int count,
 rcclResult_t rcclAllGather(const void* sendbuff, int count, rcclDataType_t datatype,
                             void* recvbuff, rcclComm_t comm, hipStream_t stream);
 
+//! Does reduction op on sendbuff on all gpus and scatter the result to all gpus
```

Comment:
scatters

Reply:
will fix this

Reply:
Please review function description patch